hbw/app/javascript/packs/hbw/components/form/datetime.js.jsx

```diff
@@ -2,75 +2,86 @@ import cx from 'classnames';
 import compose from 'shared/utils/compose';
 import { withCallbacks, withConditions, withErrorBoundary } from 'shared/hoc';
 import TranslationContext from 'shared/context/translation';
+import DatePickerWrapper from 'shared/element/date_picker_wrapper';
+import convertToDateFNSDateFormat from 'shared/utils/date_format_converter';
+import { format } from 'date-fns';
+import { initDatePickerLocale } from '../../init/date_localizer';
 
-modulejs.define('HBWFormDatetime', ['React', 'ReactDOM', 'jQuery', 'moment'], (React, ReactDOM, jQuery, moment) => {
+modulejs.define('HBWFormDatetime', ['React'], (React) => {
   class HBWFormDatetime extends React.Component {
     static contextType = TranslationContext;
 
     constructor (props, context) {
       super(props, context);
-      let defaultValue;
-      let value;
 
-      const locale = props.params.locale || context.locale.code || 'en';
-      const format = props.params.format || 'MM/DD/YYYY';
+      initDatePickerLocale();
```

Comment:
it's good but you should probably check if it's fine to register locales on every picker initialization. If it doesn't do terrible stuff inside on every run then it's good to go. Otherwise, we have to move it to top-level initialization code

Reply:
Moved it from constructor to upper scope after import so it register locale only once and not on every date input field rendered on form